go/server/handlers.go

```diff
@@ -33,8 +34,22 @@ func (s *Server) informationHandler(c *gin.Context) {
 }
 
 func (s *Server) homeHanlder(c *gin.Context) {
+	oltpData, err := macrobench.GetResultsForLastDays(macrobench.OLTP, "webhook", 31, s.dbClient)
+	if err != nil {
+		log.Println(err.Error())
+		return
+	}
+
+	tpccData, err := macrobench.GetResultsForLastDays(macrobench.TPCC, "webhook", 31, s.dbClient)
+	if err != nil {
+		log.Println(err.Error())
+		return
+	}
```

Comment:
can some error message be shown on the UI if the data is not retrieved?

Reply:
Good idea. I have added a "debug" method to render error in c8a104b0a3ae8f25b745c268bde94f166121b41c